### PR TITLE
Csld28 improve differences

### PR DESCRIPTION
**Pull request by gromakovsky**

### CODE REVIEW

File: _docs/cardano/2017-01-09-differences.md [hunk 1]
```diff
@@ -77,11 +99,22 @@ a `(1 - p₁) … (1 - pⱼ₋₁) pⱼ`-biased coin to see whether `j`-th
...
+(32-bits) seed is used for initializing PRG instead of using `n ⌈log
+λ⌉` random bits.
 
 ## Commitments, openings, shares sending
 
-Time of sending is randomized within ε.
+Time of sending is randomized within small interval. It is done to
+avoid network overload when all coin tossing participants send their
+data at the same time. This interval is chosen to be small enough for
+protocol to remain secure. If this data is sent to late and there are
```
Comment (zakharvoit) [inline _docs/cardano/2017-01-09-differences.md]: s/to late/too late/

File: _docs/cardano/2017-01-09-differences.md [hunk 2]
```diff
@@ -9,22 +9,39 @@ visible: true
...
 
 ## Time, Slots, and Synchrony
 
 In a basic model of *Ouroboros* time is divided into discrete units
 called *slots*. However, there are no details on how to obtain current
 time securely and with enough precision.
 
-In *cardano-sl* current time is obtained by querying a predefined set of
-NTP servers.
+In *cardano-sl* current time is obtained by querying a predefined set
```
Comment (zakharvoit) [inline _docs/cardano/2017-01-09-differences.md]: Above you used "Cardano SL" and here "cardano-sl"

Comment (gromakovsky) [inline _docs/cardano/2017-01-09-differences.md]: @rockbmb which version do you think is better?

Comment (rockbmb) [inline _docs/cardano/2017-01-09-differences.md]: `cardano-sl`, in this case.

**Pull request merged**